### PR TITLE
Css variables

### PR DESCRIPTION
**Pull request by markgallardo**

### CODE REVIEW

File: 19 - Webcam Fun/index.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
         <input type="range" min=0 max=255 name="bmin">
         <label for="bmax">Blue Max:</label>
         <input type="range" min=0 max=255 name="bmax">
-      </div> -->
```
Comment (BHARGAVPATEL1244) [inline 19 - Webcam Fun/index.html:32]: </div>

File: 19 - Webcam Fun/index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
   <div class="photobooth">
     <div class="controls">
       <button onClick="takePhoto()">Take Photo</button>
-<!--       <div class="rgb">
```
Comment (BHARGAVPATEL1244) [inline 19 - Webcam Fun/index.html:13]: < div class = " rgb " >

**Review by BHARGAVPATEL1244 — APPROVED** [review]

File: 01 - JavaScript Drum Kit/style.css [hunk 3]
```diff
@@ -34,8 +34,8 @@ body,html {
 
 .playing {
   transform: scale(1.1);
-  border-color: #ffc600;
-  box-shadow: 0 0 1rem #ffc600;
```
Comment (BHARGAVPATEL1244) [inline 01 - JavaScript Drum Kit/style.css:38]: border-color : #808080 ;
box-shadow : 0 0 1rem #808080 ;

File: 01 - JavaScript Drum Kit/style.css [hunk 4]
```diff
@@ -47,5 +47,5 @@ kbd {
   font-size: 1.2rem;
   text-transform: uppercase;
   letter-spacing: .1rem;
-  color: #ffc600;
```
Comment (BHARGAVPATEL1244) [inline 01 - JavaScript Drum Kit/style.css:50]: color : #663399 ;

**Review by BHARGAVPATEL1244 — APPROVED** [review]

File: 02 - JS and CSS Clock/index-START.html [hunk 5]
```diff
@@ -56,19 +56,58 @@
...
+
+    const seconds = now.getSeconds();
+    const secondsDeg = ((seconds / 60 ) * 360) + 90
+    secondHand.style.transform = `rotate(${secondsDeg}deg)`;
+     console.log(hour)
+}
 
+  setInterval(setDate,1000)
 
-  </script>
```
Comment (brijesh209) [inline 02 - JS and CSS Clock/index-START.html:72]: </script>

**Review by brijesh209 — APPROVED** [review]

**Comment by palashmon** [comment]
Hi, I think you wanted to push these changes to your own forked repo instead of this main repo. In that case, closing this PR for now. Thanks!

**Pull request closed without merging**